iceoryx_hoofs/legacy/include/iceoryx_hoofs/cxx/unique_ptr.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2022 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef IOX_HOOFS_CXX_UNIQUE_PTR_HPP
+#define IOX_HOOFS_CXX_UNIQUE_PTR_HPP
+
+#include "iceoryx_hoofs/memory/unique_ptr.hpp"
+
+namespace iox
+{
+namespace cxx
+{
+using memory::unique_ptr;
```

Comment:
@FerdinandSpitzschnueffler @elfenpiff For all public include such a legacy trampoline header has to be added in order to keep the transition pain for our users to a minimum.